samples/quickstart.py

```diff
@@ -0,0 +1,45 @@
+# This application demonstrates the usage of the Analytics Admin API using
```

Comment:
Please add a license header to the top of this file.

Reply:
Bu, that would be awesome. Let's setup noxfile/kokoro!

Reply:
done